### ISSUE TITLE
Fix CI log interleaving

### ISSUE THREAD

**Issue by spikymonkey** [spring-cloud/spring-cloud-app-broker#380]
Gradle parallelisation results in logs from different tests interleaving in CI, e.g.

```
org.springframework.cloud.appbroker.acceptance.CreateInstanceWithOAuth2CredentialsAcceptanceTest STANDARD_OUT

      .   ____          _            __ _ _
     /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
    ( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
     \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
      '  |____| .__|_| |_|_| |_\__, | / / / /
     =========|_|==============|___/=/_/_/_/
     :: Spring Boot ::       (v2.1.13.RELEASE)


org.springframework.cloud.appbroker.acceptance.CreateInstanceAcceptanceTest STANDARD_OUT

      .   ____          _            __ _ _
     /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
    ( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
     \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
      '  |____| .__|_| |_|_| |_\__, | / / / /
     =========|_|==============|___/=/_/_/_/
     :: Spring Boot ::       (v2.1.13.RELEASE)


org.springframework.cloud.appbroker.acceptance.AppManagementStopAcceptanceTest STANDARD_OUT

      .   ____          _            __ _ _
     /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
    ( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
     \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
      '  |____| .__|_| |_|_| |_\__, | / / / /
     =========|_|==============|___/=/_/_/_/
     :: Spring Boot ::       (v2.1.13.RELEASE)


org.springframework.cloud.appbroker.acceptance.CreateInstanceWithOAuth2CredentialsAcceptanceTest STANDARD_OUT
```

This makes it hard to understand what's going on with individual tests. We've fixed this in other projects recently - should be able to apply the same patterns to App Broker too.

**Comment by gberche-orange** [comment]
@spikymonkey 
The approach I've used in osb-cmdb ci is to upload junit test result reports (html) to a remote repository. The test results include standard output associated with each test, solving the interleaving traces. Could that help in your case ?

**Comment by spikymonkey** [comment]
Hi @gberche-orange - being able to access the html reports is definitely nice - they're a lot easier on the eye than stdout :) But I'd still like the default build to be as friendly as possible without requiring extra effort like this. I have a log buffering pattern already that I'm going to apply to the build which should sort this out - really just filing this issue as a reminder for myself to do the work :)

**Comment by spikymonkey** [comment]
Fixed in #387
